res/css/views/messages/_MImageBody.scss

```diff
@@ -38,6 +38,12 @@ $timeline-image-border-radius: 8px;
     // Necessary for the border radius to apply correctly to the placeholder
     overflow: hidden;
     contain: paint;
+
+    min-height: $font-44px;
```

Comment:
nit: we shouldn't use the font variables for non-font usecases, unless this is meant to be relative to a message of x lines or something?